core/controllers/classifier.py

```diff
@@ -123,3 +129,61 @@ def post(self):
         classifier_services.mark_training_job_complete(job_id)
 
         return self.render_json({})
+
+
+def update_failed_jobs(job_models):
```

Comment:
So are we just going to leave FAILED jobs as FAILED and not retry them, @anmolshkl @seanlip? I thought we were planning to resubmit the FAILED jobs?